pypeit/specobj.py

```diff
@@ -502,8 +503,8 @@ def apply_flux_calib(self, wave_zp, zeropoint, exptime, tellmodel=None, extinct_
             exptime (float):
                 Exposure time
             tellmodel (?):
-                Telluric correction
-            extinct_correct (?):
+                Telluric correction. Note: This is deprecated and will be removed in a future version.
```

Comment:
I followed the code a bit but not far enough to understand how this is deprecated.  If providing tellmodel here (and ultimately how it's used in `get_sensfunc_factor`) produces an incorrect result, we should deprecate it now.